controllers/comments-controller.js

```diff
@@ -0,0 +1,22 @@
+const {insertComment} = require('../models/comments-model');
+
+exports.addComment = (req, res, next) => {
+    const { article_id } = req.params;
+    const { username, body} = req.body
+
+
+    if(isNaN(Number(article_id))){
+        return res.status(400).send({ msg: 'invalid input'});
+    }
+
+    if (!username || !body){
+        return res.status(400).send({ msg: 'Username and body are both required'})
+    }
+
```

Comment:
You don't need to manually check the above - if any of these are missing or the wrong data type then psql will throw an error itself because they don't follow the rules of the seed. It is however not 'wrong', could just be handled a different way!